packages/ui/src/solidity/AccountControls.svelte

```diff
@@ -189,7 +188,7 @@
       <input type="radio" bind:group={opts.signer} value="MultisigWeighted" />
       Multisig Weighted*
       <HelpTooltip
-        link="https://docs.openzeppelin.com/community-contracts/0.0.1/multisig-account#multisignererc7913weighted"
+        link="https://docs.openzeppelin.com/community-contracts/0.0.1/multisig#multisignererc7913weighted"
```

Comment:
nit: we can remove the version number from the link
```suggestion
        link="https://docs.openzeppelin.com/community-contracts/multisig#multisignererc7913weighted"
```